crates/red_knot_python_semantic/resources/mdtest/sys_version_info.md

```diff
@@ -1,5 +1,10 @@
 # `sys.version_info`
 
+```toml
+[environment]
+target-version = "3.9"
+```
```

Comment:
hmm... not the end of the world, but on the red-knot CLI we set a target version using `--target-version=py39` rather than `--target-version="3.9"`. Similarly for Ruff, the option is `--target-version=py39` and in a configuration file it's `target-version = "py39"`.

I don't actually have a strong opinion on which is better, but it'd be nice to keep the red-knot CLI and the configuration setting here in sync, or it's hard to remember the difference between the two.

Reply:
uv uses `3.9` and I'd prefer that format too

Reply:
I see. In that case I'm fine with this, but we should open an issue to change how the CLI works.

Reply:
I plan to scope out the CLI work once the proposal is in a good shape

Reply:
However it ends up, we need to make sure that we remember to align the two though

Reply:
Absolutely. Consider me responsible for it

Reply:
Absolutely. Consider me responsible for it